composeApp/src/commonMain/kotlin/com/github/familyvault/models/chat/ChatThread.kt

```diff
@@ -1,13 +1,15 @@
 package com.github.familyvault.models.chat
 
 import com.github.familyvault.models.enums.chat.ChatThreadType
+import com.github.familyvault.models.enums.chat.ChatIconType
 
 data class ChatThread(
     val id: String,
     val name: String,
     val participantsIds: List<String>,
     val lastMessage: ChatMessage?,
     val type: ChatThreadType,
+    val icon: ChatIconType? = null
```

Comment:
To nie powinno się nazywać `icon`. Potem będziesz się do tego odwoływał `icon.icon`. Nazwij to `iconType`